### ISSUE TITLE
behavior after a write error

### ISSUE THREAD

**Issue by jkbzh** [pchampin/minutes_to_gh#6]
I think that if the script encounters an  error while writing to github (like token expired, github requests x hour attained..), the script will return the error and finish.

Would be good to know what would be the plan B in this case, like being able to invoke the bot and tell it to run against the stored minutes.

If you already took this into consideration, please feel free to delete this issue.

**Comment by pchampin** [comment]
> I think that if the script encounters an error while writing to github (like token expired, github requests x hour attained..), the script will return the error and finish.

I agree. The errors I encountered, which prompted me to recover and continue posting, were buggy GitHub URLs in the minutes, that where preventing the following (correct) URLs to be processed. But I agree that the kind of errors you mention should cause the bot to stop completely...

**Comment by pchampin** [comment]
I looked at this issue, but FTR, differentiating the different kinds of Github errors is in fact not trivial with the [octocrab](https://docs.rs/octocrab/latest/octocrab/) library.
